src/main/kotlin/no/fdk/dataset_catalog/rdf/RDFDatasetUtils.kt

```diff
@@ -28,8 +29,8 @@ fun Resource.createDatasetResource(ds: Dataset, datasetUri: String): Resource {
                     .safeAddLiteralByLang(DCATNO.objective, ds.objective)
                     .addContactPoints(ds.contactPoint)
                     .safeAddLangListProperty(DCAT.keyword, ds.keyword)
-                    .safeAddDateTimeLiteral(DCTerms.issued, ds.issued)
-                    .safeAddDateTimeLiteral(DCTerms.modified, ds.modified)
+                    .safeAddDateTimeLiteral(DCTerms.issued, ds.issued, XSDDatatype.XSDdateTime)
+                    .safeAddDateTimeLiteral(DCTerms.modified, ds.modified, XSDDatatype.XSDdate)
```

Comment:
Hvorfor to forskjellige XSD typer? Kan være like greit å bruke `XSDdateTime` for begge to

Reply:
Dette er for å samsvare med gamle dataset-catalogue. Er ikke sikker på om det har noe å si for harvesting @NilsOveTen?

Reply:
Høsteren gir faen. Parseren bryr seg mer, men den takler fint både date og datetime. Bruk det som fungerer best